Update copy on WPAdmin account toggle

getdave opened the pull request:
Fixes https://github.com/Automattic/wp-calypso/issues/50202


#### Changes proposed in this Pull Request

* Update copy on WPAdmin toggle under Account Settings to better reflect intent of toggle.

#### Testing instructions

<!--
Add as many details as possible to help others reproduce the issue and test the fix.
"Before / After" screenshots can also be very helpful when the change is visual.
-->

* Open Calypso Live link.
* Go to Account section `/me/account`.
* See new copy on toggle.

##### Please note

* The PR doesn't match the design because the design isn't inline with the current Calypso interface.
* I've omitted the support docs link as we don't yet have the link available.

<!--
Link a related issue to this PR. If the PR does not immediately resolve the issue,
for example, it requires a separate deployment to production, avoid
using the "fixes" keyword and instead attach the [Status] Fix Inbound label to
the linked issue.
-->

getdave committed:
Update copy

getdave committed:
Add temp link to support docs

getdave committed:
Remove link as we don’t have it available yet.

cpapazoglou commented on client/me/account/main.jsx:
```diff
@@ -1057,15 +1057,21 @@ class Account extends React.Component {
+										'Enabling this will replace your dashboard pages with more advanced wp-admin equivalents when possible.'
 									) }
+
+									i18n.translate( 'I feel {{a}}very{{/a}} strongly about this.', {
+										components: {
+											a: <a />
+										}
```
Something goes wrong here 😜!

getdave replied:
Ha! Good spot :)

cpapazoglou reviewed (changes requested):
❤️ This is failing to build for me ❤️!

Commit:
Removes errant code.

cpapazoglou commented:
It works as expected, I will leave it up to Saul for the final call!

ghost commented:
> The PR doesn't match the design because the design isn't inline with the current Calypso interface.

Apologies - working fast and must have added an old mock-up.

<img width="741" alt="Screen Shot 2021-03-01 at 11 20 44 AM" src="https://user-images.githubusercontent.com/67021691/109526063-2c25a800-7a80-11eb-9448-d9e9f170a51c.png">

I'm not sure about making "show advanced dashboard pages" as the section label. It becomes a little disassociated with the toggle action. 

<img width="996" alt="Screen Shot 2021-03-01 at 11 26 37 AM" src="https://user-images.githubusercontent.com/67021691/109526822-fdf49800-7a80-11eb-9dd2-6c0f8bf8c82e.png">

Can it be placed back into the toggle with the description text, like so?

getdave committed:
Update wording and tweak visual styles accordingly.

Resolves https://github.com/Automattic/wp-calypso/pull/50588#issuecomment-788084409

getdave commented:
> Can it be placed back into the toggle with the description text, like so?

Yes it can! Done. I hope that's more what you were expecting?

ghost commented:
>  I hope that's more what you were expecting?

Yeah, looking good! Thanks.

getdave commented:
Merging based on https://github.com/Automattic/wp-calypso/pull/50588#pullrequestreview-600903831
